### PR TITLE
[T2A2][T11-A2]Joshua Ching

### PR DESCRIPTION
**Pull request by JoshuaChing**

### CODE REVIEW

**Comment by JoshuaChing** [comment]
Ready for review

**Comment by JoshuaChing** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1153,14 +1153,14 @@ private static String getUsageInfoForExitCommand() {
...
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, EMPTY_STRING);
+    private static String removePrefix(String fullString, String prefix) {
```
Comment (nirandiw) [inline src/seedu/addressbook/AddressBook.java:1166]: This implementation is not abstract enough to remove only the prefix. Consider revising.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1164,7 +1164,7 @@ private static String getUsageInfoForExitCommand() {
      * @return  string without the prefix
      */
     private static String removePrefix(String fullString, String prefix) {
-        return fullString.replace(prefix, EMPTY_STRING);
+        return fullString.replaceFirst(prefix, EMPTY_STRING);
```
Comment (nirandiw) [inline src/seedu/addressbook/AddressBook.java:1167]: This revision still does not remove the value of prefix , from the beginning of the 'fullString' variable. 
The method should replace the value only if the value of prefix is a prefix of the value of fullString.

**Review by nirandiw — COMMENTED** [review]
@JoshuaChing Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**